### PR TITLE
Prettify timeline 3

### PR DESCRIPTION
**Pull request by Sorien**
Fix timeline comments last section borders

image

Fix DropZone and buttons left alignment 

image

Update review comment form to look the same as others 

image

Fix Time Tracker Add Time button left border

image

Remove/Tidy up some classes

### CODE REVIEW

File: web_src/less/_repository.less [hunk 1]
```diff
@@ -1045,6 +1042,17 @@
                     box-shadow: none;
                 }
             }
+
+            .ui.form {
+                .field.footer {
+                    overflow: hidden;
+                    .controls.right {
```
Comment (lafriks) [inline web_src/less/_repository.less]:
```suggestion
                    .controls.right :last-child {
```

Comment (Sorien) [reply web_src/less/_repository.less]: I'm not sure if it's the right approach, will probably switch buttons margin from right to left side when parent container has `right` class

Comment (lafriks) [reply web_src/less/_repository.less]: It is the same as yours, no need to use stacked syntax if there is single rule

Comment (Sorien) [reply web_src/less/_repository.less]: I was just talking to myself while reading those styles.

**Review by silverwind — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Comment by lunny** [comment]
Is this still a draft?

**Comment by 6543** [comment]
a draft with lgtm 😆

**Comment by Sorien** [comment]
I wanted to try if it's possible to align buttons by switching margin from right to left, I will try to look at it tomorrow.

**Comment by Sorien** [comment]
ok should be ready, rebased commits (sorry for that), new commits from 
41a427a

File: web_src/less/_repository.less (hunk 1 above)
Comment (Sorien) [reply web_src/less/_repository.less]: changed in https://github.com/go-gitea/gitea/pull/11139/commits/41a427a3351b5609aa905c9647a9f2f555c01788

**Comment by Sorien** [comment]
I have a question why there is file dropzone area in the preview tab?

**Review by 6543 — CHANGES_REQUESTED** [review]
I need a look again

**Comment by guillep2k** [comment]
@Sorien lint fails.

**Review by 6543 — APPROVED** [review]

**Comment by guillep2k** [comment]
@silverwind Care to take another look?

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
> I have a question why there is file dropzone area in the preview tab?

Not sure what you mean but I'd say a dropzone should be outside the write/preview tabs.

**Pull request merged**